common/network-common/src/main/java/org/apache/spark/network/protocol/StreamChunkId.java

```diff
@@ -26,14 +26,7 @@
 /**
 * Encapsulates a request for a particular chunk of a stream.
 */
-public final class StreamChunkId implements Encodable {
```

Comment:
Although this is `public`, this is not a part of our public Java doc, right?

- https://spark.apache.org/docs/latest/api/java/index.html

Reply:
Yes